lib/browser/mqtt.ts

```diff
@@ -100,6 +102,20 @@ export interface MqttConnectionConfig {
      */
     protocol_operation_timeout?: number;
 
+    /**
+     * Minimum seconds to wait between reconnect attempts.
+     * Must be <= {@link reconnect_max_sec}.
+     * Wait starts at min and doubles with each attempt until max is reached.
+     */
+    reconnect_min_sec?: number;
```

Comment:
The intent of the default discussion from yesterday was to see if we can drop the nullable property here and initialize the values to the defaults.  Since client creation requires this, then the config init is the only place we'd need to initialize to the defaults.

Reply:
Javascript does not allow a default in interface, that's why we set up the default value in MqttConnection.